RACK-Ontology/ontology/REVIEW.sadl

```diff
@@ -29,14 +38,14 @@ REVIEW_STATE (note "Life-cycle status of a review log")
     is a type of THING,
     must be one of {Passed, ReviseWithoutReview, ReviseWithReview}.
 Passed              is a REVIEW_STATE with identifier "Passed".
-ReviseWithoutReview is a REVIEW_STATE with identifier "Review Without Review".
+ReviseWithoutReview is a REVIEW_STATE with identifier "Revise Without Review".
 ReviseWithReview    is a REVIEW_STATE with identifier "Revise With Review".
 
 REVIEW
-	(note "Code review ACTIVITY")
+	(note "Artifact review ACTIVITY")
 	is a type of ACTIVITY.
 
-	author (note "AGENT(s) putting items up for review") describes REVIEW with values of type AGENT.
+	author (note "AGENT(s) putting items up for review") describes REVIEW with a single value of type AGENT.
```

Comment:
while in the norm I would agree the cardinality would be 1, I am not sure I this would always be the case.  I would suggest to just leave this cardinality unrestricted.  If we do keep the restriction then the note should be updated to remove the "(s)" from AGENT